src/go/k8s/apis/core/v1alpha1/redpandacluster_types.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package v1alpha1 represent Custom Resource definition of the vectorized.io core group
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RedpandaClusterSpec defines the desired state of RedpandaCluster
+type RedpandaClusterSpec struct {
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Version of Redpanda to use from upstream.
+	Version	string	`json:"version,omitempty"`
+	// Replicas determine how big the cluster will be.
+	// +kubebuilder:validation:Minimum=0
+	Replicas	*int32	`json:"replicas,omitempty"`
+	// Resources 1 Redpanda container will use.
+	// To calculate overall resource consumption one need to
+	// multiply replicas against limits
+	Resources	corev1.ResourceRequirements	`json:"resources"`
+	// Configuration represent redpanda specific configuration
+	Configuration	RedpandaConfig	`json:"configuration,omitempty"`
+}
+
+// RedpandaClusterStatus defines the observed state of RedpandaCluster
+type RedpandaClusterStatus struct {
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Replicas show how many nodes are working in the cluster
+	// +optional
+	Replicas	int32	`json:"replicas,omitempty"`
```

Comment:
I guess this is OK for now but we'll need to change this in the future to provide better status.

Reply:
Agree!